ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/DomainSpecific/Paged/FrameCheckWrapper.js

```diff
@@ -3,18 +3,16 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-'use strict'
```

Comment:
Why are you removing strict mode instead of just fixing its definition?

Reply:
And if the issue is that we're missing a semicolon on `use strict` then we're missing it in a number of other scripts as well: `BraveGetUA.js`, `gpc.js`, `SiteStateListener.js`, `cosmetic-filter-tests.js`, `farbling-tests.js`, `request-blocking-tests.js`

Reply:
I removed the semicolon changes. Sorry this was sort of experimental changes that I should have removed and tested. 
I did remove the `strict mode` flag though because it doesn't seem like the engine script uses strict mode.